src/main/scala/com/fulcrumgenomics/vcf/filtration/FilterSomaticVcf.scala

```diff
@@ -52,17 +52,40 @@ import scala.collection.immutable.ListMap
+    |in with one or more adenines during A-base addition. This presents specifically as errors to T at
+    |the beginning of reads (and in very short templates, as matching errors to A at the ends of reads).
+    |
+    |The filter adds the `INFO` field `ATP` to SNVs with an A or T alternate allele. This field records
+    |the p-value for rejecting the possibility that the variant is an A-tailing artifact. If a threshold
+    |p-value is specified, then the `FILTER` tag `ATailingArtifact` will be applied to variants with
+    |p-values below the threshold.
+    |
+    |Two options are available:
+    |
+    |* `--a-tailing-distance`  allows control over how close to the ends of reads/templates errors can be
+    |                          considered to be candidates for the A-tailing artifact. Higher values
+    |                          decrease the power of the test, so this should be set as low as possible
+    |                          given observed errors.
+    |* `--a-tailing-p-value`   the p-value below which a filter should be applied. If no value is supplied
+    |                           only the annotation is produced and no filtering is performed.
+    |
     |### End Repair Artifact Filter
     |
-    |The end repair artifact filter attempts to measure the probability that a variant is caused by
-    |errors in the template generated during the end-repair and A-base addition steps that are common
-    |to many Illumina library preparation protocols.  The artifacts occur if/when the end repair creates
-    |a recessed 3' end which is subsequently and incorrectly filled in with As during A-base addition.
-    |This presents specifically as errors to T at the beginning of reads (and in very short templates,
-    |as matching errors to A at the ends of reads).
+    |The end repair artifact filter attempts to measure the probability that a SNV is the product of
+    |errors in the template generated during the end repair step that is common to many Illumina library
+    |preparation protocols, in which single-stranded 3' overhands are filled in to create a blunt end.
+    |These artifacts originate from single-stranded templates containing damaged bases, often as a
+    |consequence of chemical/oxidative stress. These DNA lesions, for example 8-oxoguanine, undergo
+    |mismatched pairing, which after PCR appear as mutations at the ends of reads.
     |
-    |The filter creates the `ERAP` info attribute on SNVs with an A or T alternate allele, to record
-    |the p-value for rejecting the possibility that the variant is due to an end repair artifact.
+    |The filter adds the `INFO` field `ERP` to SNVs. This field records the p-value for rejecting the
+    |possibility that the variant is an end repair artifact. If a threshold p-value is specified, then
+    |the `FILTER` tag `EndRepairArtifact` will be applied to variants with p-values below the threshold.
```

Comment:
Just below (<), or below and equal to (<=)?

Reply:
```suggestion
    |the `FILTER` tag `EndRepairArtifact` will be applied to variants with p-values less than or equal to the threshold.
```